reports/Delaware.md

```diff
@@ -2,7 +2,9 @@
 
 ### USA Today journalist arrested | June 9th
 
-A journalist is filming and asking the police why protestors are being arrested. The police tackle the him and he states loudly "I'm with the press" multiple times. The video ends with an officer confiscating his press press and bag; another officer off camera is heard saying to the journalist "Shh, you've been no help here". The news article video has more context, the reddit video only shows the arrest. 
+A journalist is filming and asking the police why protestors are being arrested. The police tackle him and he states loudly "I'm with the press" multiple times. The video ends with an officer confiscating his press press and bag; another officer off camera is heard saying to the journalist "Shh, you've been no help here". The news article video has more context, the reddit video only shows the arrest. 
```

Comment:
The video ends with an officer confiscating his press press and bag; ->
The video ends with an officer confiscating his press ? badge ? and bag;